### PR TITLE
Adds resize variable for mobs

### PR DESCRIPTION
**Pull request by AnturK**
Resize variable to make transform scaling easier for badminnery.

space station 13 2015-05-02 213319

### CODE REVIEW

**Comment by Xhuis** [comment]
i've fought mudcrabs tougher than you

File: code/modules/mob/living/carbon/update_icons.dm [hunk 1]
```diff
@@ -20,6 +20,10 @@
 					final_dir = pick(NORTH, SOUTH) //So you fall on your side rather than your face or ass
 
 		lying_prev = lying	//so we don't try to animate until there's been another change.
+	if(resize != 1)
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/update_icons.dm]: replace ```1``` with ```RESIZE_DEFAULT_SIZE```

**Comment by RemieRichards** [comment]
This is good :+1:

File: code/modules/mob/mob_defines.dm [hunk 2]
```diff
@@ -152,4 +152,6 @@
 	var/turf/listed_turf = null	//the current turf being examined in the stat panel
 
 	var/list/permanent_huds = list()
-	var/permanent_sight_flags = 0
\ No newline at end of file
+	var/permanent_sight_flags = 0
+
+	var/resize = 1 //Badminnery resize
```
Comment (RemieRichards) [inline code/modules/mob/mob_defines.dm]: hnnng. stick the define in please.

Comment (AnturK) [inline code/modules/mob/mob_defines.dm]: Ah forgot this one

**Comment by Niknakflak** [comment]
Oh boy, cant wait to make goatsie, polysie, Iansie and everything inbetween

**Comment by PKPenguin321** [comment]
>goatsie

**Comment by Jordie0608** [comment]
I am reminded of #2883

**Comment by Miauw62** [comment]
made me laugh :+1:

**Comment by Alek2ander** [comment]
:-1: Mobs
:+1: Everything

**Pull request merged**